### ISSUE TITLE
Base R plotting errors due to margins

### ISSUE THREAD

**Issue by ethanwhite** [coatless/quarto-webr#1]
To reproduce quarto preview the following file:

    ---
    title: plot
    filters:
      - webr
    ---

    ```{r}
    plot(mtcars$hp, mtcars$mpg)
    ```

    ```{webr}
    plot(mtcars$hp, mtcars$mpg)
    ```

Result:

The `{r}` block plots correctly. The `{webr}` block returns `Error: figure margins too large`

Thanks for this amazing extension! I'm already working on applying it to educational material and it really is fantastic! Working here if you're interested https://github.com/ethanwhite/datacarp-interactive. Demo site: https://interactive-lessons.weecology.org/

**Comment by coatless** [comment]
Thanks, this should be fixed now in 1a1750d69536693e73dffb917886dbf7c825532f. 
<img width="1649" alt="show-fixed-margin-graph" src="https://user-images.githubusercontent.com/833642/224571408-15bcee0d-eefb-49ea-b9b8-917d2a69091f.png">

**Comment by ethanwhite** [comment]
Works great! Thanks!
